calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
My original objection to the term "pulling selectors" wasn't the term pulling - it was the term selectors.  Calico uses both the terms "labels" and "selectors" in its policy model.  What we are pulling from the container and applying to the endpoints are the "labels".  The "selectors" are defined on the policy resources.

I'm happy to follow Matts suggestion about talking about traffic and explicit allowing it through calico policy.  If it can be done succinctly I think mentioning that profiles will not be used to enforce network isolation that would also be good - but at the very least we should at least be going into that level of detail in our documentation.